package.json

```
@@ -6,7 +6,9 @@
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "testGUI": "npx cypress open",
```

O `npx` não é necessário.
```suggestion
    "testGUI": "cypress open",
```